steps/24_categorical_support/step.md

```diff
@@ -0,0 +1,143 @@
+# Adding categorical support to sktime
+
+Contributors: @Abhay-Lejith
+
+## Introduction
+
+### Objectives 
+Allow users to pass data with categorical features to any estimator. If not supported, return an informative error message. Otherwise it should run without errors.
```

Comment:
This objective already assumes a design. "if not supported, ..."

Try to separate objective from solutioning, please.

Reply:
Yes, I didn't notice, will change it.